R/hello.R

```diff
@@ -26,10 +26,12 @@ hipercow_hello <- function(progress = NULL, timeout = NULL, root = NULL) {
   ## TODO: here we also want to modify the template to use the
   ## fast-but-short queue; we can't do that until mrc-4801 is done
   ## though.
-  id <- task_create_expr(utils::sessionInfo(), submit = TRUE, root = root)
+  id <- task_create_expr(hipercow:::hipercow_handshake(),
+                         submit = TRUE, root = root)
```

Comment:
```
moo <- read_string(hipercow_file("comms/moo"))
task_create_expr({
  message(moo)
  "Moo"
}, submit = TRUE, root = root)
```


with comms/moo containing

```
 -----
Moooooo!
 ------
    \   ^__^
     \  (oo)\ ________
        (__)\         )\ /\
             ||------w|
             ||      ||
```

and two helper functions (which exist already in windows)